### ISSUE TITLE
"Add" "Remove" etc. UI elements not rendered in editor.

### ISSUE THREAD

**Issue by SnowblindFatal** [NathanWarden/ai-behavior#1]
The "+", "-" and such symbols are not rendered. Clicking on the empty spaces where they should be seems to work, but using the UI in this way is... not optimal. :D Anyway, I took a quick dive into the code and I'm not entirely sure where to start looking. Haven't worked with the Unity Editor GUI system to know what could cause this kind of issue. No warnings generated by the AIVehavior seems to be related to this. 

![issue2](https://user-images.githubusercontent.com/1068367/56434054-bb39d480-62db-11e9-8b11-0c80af08d186.jpg)
![issue1](https://user-images.githubusercontent.com/1068367/56434056-bd039800-62db-11e9-9cd6-7b170840a54e.jpg)

Using the version 2019.1.0f2

**Comment by NathanWarden** [comment]
I don't have 2019 installed yet. Is this after AI Behavior is copied into a separate project? If the directory structure isn't the same as this repo then AIBehaviorsStyles.cs has a path variable you can change at the bottom of the script that should fix the problem.

If the directory structure is the same let me know and I'll go ahead and see if there's something with 2019 that messes it up.

Thanks!

**Comment by SnowblindFatal** [comment]
Oh yes, that did it. Somehow I didn't notice the path variable when I was quickly checking the image loading and since there was no error message, I wasn't sure what was up. I have my assets organised so that the ones I haven't made are under the folder "Imported". 

Thanks for the tip, all works now. I suppose relative paths are somehow problematic?

**Comment by NathanWarden** [comment]
Yeah, I wrote that code many years ago and haven't given it a second look yet. But, I'm sure there's probably a simple solution and just need to give it a little bit of thought.

I'll leave this open until I do.

Thanks!
